common/unified-chart/values.yaml

```diff
@@ -11,9 +11,29 @@ nameSuffix: v2
 
 argoRollouts:
   enabled: true
-  activeMetadata:
-    labels:
-      role: active
+  # activeMetadata:
+  #   labels:
+  #     role: active
+  progressiveDelivery:
+    enabled: true
+    stableMetadata:
+      labels:
+        role: stable
+    canaryMetadata:
+      labels:
+        role: canary
+    blackBox:
+      url: http://nexus.default.svc.cluster.local:8080/blackbox
```

Comment:
can we move nexus svc to port 80? all other services are aligned to 80 as the main port